supabase/migrations/20240425082036_actor_profile_id_in_audit_log_table.sql

```diff
@@ -0,0 +1,11 @@
+alter table "public"."audit_log" add column "actor_profile_id" uuid;
+
+alter table "public"."audit_log" add constraint "audit_log_actor_profile_id_fkey" FOREIGN KEY ("actor_profile_id") REFERENCES "public"."profiles"("primary_key") ON DELETE CASCADE;

```

Comment:
let's use ON DELETE SET NULL, just in case. This means `actor_profile_id`  needs to be nullable, but I think that's better than an accidental removal of a profile deleting all the logs for that profile.

Reply:
Ok makes sense. Do I need to remove the not null constraint? I didn't, and tried resetting the database, and everything worked fine - the column is not nullable but has "ON DELETE SET NULL". Not sure if it will work or not?

Reply:
ignore me i answered this by trying to delete a profile with associated audit logs - supabase errored - so i'll remove the line in the migration file that sets the null constraint